style.css

```diff
@@ -0,0 +1,153 @@
+*{
+    box-sizing: border-box;
+}
+body{
```

Comment:
Can u use margin 0 into universal selector if u use then it will apply automatically into all html tag h1 h2 etc... otherwise it will only apply into body tag

Reply:
ok,